lib/td/command/job.rb

```diff
@@ -143,6 +143,10 @@ def job_show(op)
       exclude = b
     }
 
+    op.on('-n STRING', '--null STRING', "null expression in csv or tsv (default: null)") {|s|
+      render_opts[:null_expr] = s.to_s
+    }
+
```

Comment:
I'm not pretty know optparse but can we only define and use '--null' because file_reader.rb only allows it as well?